manifests/s/SomePythonThings/WingetUIStore/1.3.1/SomePythonThings.WingetUIStore.locale.en-US.yaml

```
@@ -0,0 +1,36 @@
+# Created with YamlCreate.ps1 v2.1.3 using InputObject 🤖 $debug=QUSU.7-2-6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: SomePythonThings.WingetUIStore
+PackageVersion: 1.3.1
+PackageLocale: en-US
+Publisher: SomePythonThings
```

```suggestion
Publisher: Martí Climent
```
This must match Control Panel.